src/v/features/feature_table.cc

```diff
@@ -55,6 +55,8 @@ std::string_view to_string_view(feature f) {
         return "seeds_driven_bootstrap_capable";
     case feature::tm_stm_cache:
         return "tm_stm_cache";
+    case feature::handling_isolated_node:
+        return "handling_isolated_node";
```

Comment:
the tense of this is a bit odd, maybe something like `node_isolation` would fit in better as a feature name